### ISSUE TITLE
How to visualize attention logits?

### ISSUE THREAD

**Issue by OStars** [mit-han-lab/streaming-llm#82]
Thanks for your great job! 

I have a slightly off-topic question: Could you please tell me how to visualize attention logits like `Figure 2` in the paper? I want to use this method to analyze other problems.

Hopping for your reply!

**Comment by DerrickYLJ** [comment]
Hi, have you figured out how to visualize the attention logits in the paper?
